docs/src/pages/system/styled/styled.md

```diff
@@ -93,3 +93,56 @@ const styled = createStyled({ defaultTheme });
 
 export default styled;
 ```
+
+## Difference with the `sx` prop
+
+We want the `styled` function to behave similarly to the `styled` function of other technologies, like Emotion or Styled Components. 
+The `sx` prop however, is a new way of styling your application, focused on fast customization. `styled` is a function, while `sx` is a prop of Mui components.
+
+Therefore, you may observe a few differences:
+- `sx` provides more shortcuts than `styled`
+- the style definition slightly varies
+- patterns to use props are different
```

Comment:
Would be great if we can add a short code snippet below each of the points

Reply:
I would then move the code sample upper. Text would become:

---

Therefore, you may observe a few differences:
- `sx` provides more shortcuts than `styled`
With `styled`:
```js
const MyStyledButton = styled("button")({
  mx: 1, // ❌ don't use this! This shortcut is only provided by the `sx` prop
})
```
With `sx`:
```js
const MyStyledButton = (props) => (
  <button sx={{
    mx: 1, // ✔️ this shortcut is specific to the `sx` prop,
  }}>
     {props.children}
  </button>
})
```
- the style definition slightly varies
With `styled`:
```js
const MyStyledButton = styled("button")({
  padding: 1, // means "1px", NOT "theme.spacing(1)"
})
```

With `sx`:
```js
const MyStyledButton = (props) => (
  <button sx={{
    padding: 1 // means "theme.spacing(1)", NOT "1px"
  }}>
     {props.children}
  </button>
})
```
- patterns to use props are different
With `styled`:
```js
const MyStyledButton = styled("button")({
  backgroundColor: props => props.myBackgroundColor
})
```
With `sx`:
```js
const MyStyledButton = (props) => (
  <button sx={{backgroundColor: props.myCustomColor}}>
     {props.children}
  </button>
})